pkg/utils/ocm.go

```diff
@@ -143,7 +144,14 @@ func ApplyFilters(ocmClient *sdk.Connection, filters []string) ([]*cmv1.Cluster,
 
 // GenerateQuery returns an OCM search query to retrieve all clusters matching an expression (ie- "foo%")
 func GenerateQuery(clusterIdentifier string) string {
```

Comment:
Great work with these regexes - do you mind writing a couple of simple unit test cases for this function?

Reply:
test added